node/src/command.rs

```diff
@@ -366,18 +347,7 @@ pub fn run() -> Result<()> {
 			})
 		}
 
-		#[cfg(feature = "try-runtime")]
-		Some(Subcommand::TryRuntime(cmd)) => {
-			type HostFunctions = (
-				sp_io::SubstrateHostFunctions,
-				frame_benchmarking::benchmarking::HostFunctions,
-			);
-			construct_async_run!(|components, cli, cmd, _config, runtime| { Ok(cmd.run::<Block, ()>()) })
-		}
-		#[cfg(not(feature = "try-runtime"))]
-		Some(Subcommand::TryRuntime) => Err("Try-runtime was not enabled when building the node. \
-			You can enable it with `--features try-runtime`."
-			.into()),
+		Some(Subcommand::TryRuntime) => Err("The `try-runtime` subcommand has been migrated to a standalone CLI (https://github.com/paritytech/try-runtime-cli). It is no longer being maintained here and will be removed entirely some time after January 2024. Please remove this subcommand from your runtime and use the standalone CLI.".into()),
```

Comment:
Should we follow the suggestion and remove the command entirely?

Reply:
let's make it